app/js/components/Home.jsx

```diff
@@ -17,37 +18,40 @@ export default class App extends React.Component {
       <div className="container">
 
         <div className="list-group home-list">
-          <a href="#" className="list-group-item list-group-item-action flex-column align-items-start active">
+          <Link to="#" className="list-group-item list-group-item-action flex-column align-items-start active">
             <div className="d-flex w-100 justify-content-between">
             
             <h5 className="mb-1">The Platform is running</h5><img src={(loadingGif)} alt=""/>
             </div>
             
-          </a>
-          <a href="#" className="list-group-item list-group-item-action flex-column align-items-start">
+          </Link>
+          <Link to="#" className="list-group-item list-group-item-action flex-column align-items-start" onClick={(event) => {
+            event.preventDefault();
+            window.location = '/openmrs-standalone/owa/openmrs-addonmanager/index.html';
```

Comment:
As for the above url, what if it not the standalone version?

Reply:
I have updated the PR